test/utils.test.js

```diff
@@ -11,6 +13,16 @@ describe('Util', () => {
       expect(outgoing.foo).to.equal(3)
     })
 
+    it('should parse a boolean', () => {
+      const incoming = { 'true': 'true', 'false': 'false' }
+      const spec = { true: { type: 'boolean' }, false: { type: 'boolean' } }
+      const outgoing = parseParams(incoming, spec)
+
+      expect(outgoing).to.be.an('object')
+      expect(outgoing.true).to.equal(true)
+      expect(outgoing.false).to.equal(false)
```

Comment:
The default isn't being honored if the input is neither "true" nor "false"

```suggestion
      const incoming = { 'true': 'true', 'false': 'false', 'default': 'fladeedle' }
      const spec = { true: { type: 'boolean' }, false: { type: 'boolean' }, default: { type: 'boolean', default: false } } 
      const outgoing = parseParams(incoming, spec)

      expect(outgoing).to.be.an('object')
      expect(outgoing.true).to.equal(true)
      expect(outgoing.false).to.equal(false)
      expect(outgoing.default).to.equal(false)
```